docs/book/helpers/advanced-usage.md

```diff
@@ -3,11 +3,12 @@
 ## Registering Helpers
 
 `Laminas\View\Renderer\PhpRenderer` composes a *plugin manager* for managing
-helpers, specifically an instance of `Laminas\View\HelperPluginManager`, which
-extends `Laminas\ServiceManager\AbstractPluginManager`, which is itself an
-extension of `Laminas\ServiceManager\ServiceManager`.  `HelperPluginManager` is a
-specialized service manager, so you can register a helper/plugin like any other
-service (see the [Service Manager documentation](https://docs.laminas.dev/laminas-servicemanager/configuring-the-service-manager/)
+helpers, specifically an instance of `Laminas\View\HelperPluginManager`,
+which extends `Laminas\ServiceManager\AbstractPluginManager`, which is itself
+an extension of `Laminas\ServiceManager\ServiceManager`.
+`HelperPluginManager` is a specialized service manager, so you can register a
+helper/plugin like any other service (see the
+[Service Manager documentation](https://docs.laminas.dev/laminas-servicemanager/configuring-the-service-manager/)
```

Comment:
This wrong because we are using the [one sentence per-line method](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line).

Reply:
@froschdesign I think at this point, the goal is to get basic markdown linting passing, which it isn't currently; updating to follow the current (evolving) guidelines is kind of out-of-scope.